--- mentohust-SYSU/src/myconfig.c	2017-08-30 00:59:10.063001485 +0800
+++ mentohust-SYSU-v/src/myconfig.c	2017-09-05 13:54:55.771592498 +0800
@@ -10,7 +10,7 @@
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #else
-static const char *VERSION = "0.3.1";
+static const char *VERSION = "0.4.2";
 static const char *PACKAGE_BUGREPORT = "http://code.google.com/p/mentohust/issues/list";
 #endif
 
@@ -45,9 +45,9 @@
 #else
 static const char *D_DHCPSCRIPT = "dhclient";	/* 默认DHCP脚本 */
 #endif
-static const char *CFG_FILE = "/etc/mentohust.conf";	/* 配置文件 */
-static const char *LOG_FILE = "/tmp/mentohust.log";	/* 日志文件 */
-static const char *LOCK_FILE = "/var/run/mentohust.pid";	/* 锁文件 */
+static const char *CFG_FILE = "/etc/mentohust-sysu.conf";	/* 配置文件 */
+static const char *LOG_FILE = "/tmp/mentohust-sysu.log";	/* 日志文件 */
+static const char *LOCK_FILE = "/var/run/mentohust-sysu.pid";	/* 锁文件 */
 #define LOCKMODE (S_IRUSR | S_IWUSR | S_IRGRP | S_IROTH)	/* 创建掩码 */
 
 #ifndef NO_NOTIFY
